lib/anyt/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Anyt
-  VERSION = "1.2.3"
+  VERSION = "1.2.4"
```

Comment:
Note: never touch `version.rb` in PRs. You have no idea when and how it would be released.

Reply:
I know, I had never touched it, so I especially wrote in the PR because the change is both trivial and critical. I thought it would help you. Thanks for the reminder!